lib/web/mage/backend/validation.js

```diff
@@ -25,6 +25,14 @@ define([
                     $(this.errorList.length && this.errorList[0].element || [])
                         .focus()
                         .trigger('focusin');
+
+                        $('[data-toggle=dropdown].active').trigger('open.dropdown');
+                        $('.section-config').each(function(){
+                            if($(this).find('input.mage-error').length && !$(this).hasClass("active")) {
+                            $(this).find("a[id$='-head']").click();
+                                $(this).find("input.mage-error:text:visible:first").focus();
+                            }
+                        });
```

Comment:
If I understand correctly you are iterating over all `.section-config` elements in order to find the one in which the error occurs. Would it be possible to go from `this.errorList[0].element` up in the DOM tree and check if any action is needed? Also I believe that it already contains the same value as `$(this).find("input.mage-error:text:visible:first")`.

Reply:
Hello @krzksz 
we need to check all section because there is a possibility to an error in multiple sections,
![section-config-validation](https://user-images.githubusercontent.com/43946827/69897090-dfc75600-136c-11ea-84c9-1030aa56bdbc.png)

if an error in a different section, so we need to open all section which has an error.

Reply:
Ok, now I understand, can you please add a comment for this section that explains what it does?